### ISSUE TITLE
Android 10

### ISSUE THREAD

**Issue by Renegade-Master** [Merith-TK/devdocs_webapp_kotlin#2]
Works perfectly online, but does not work offline similar to the Android 9 issue.

**Comment by Merith-TK** [comment]
Yeah I have no clue why this is, and haven't really been developing it...

**Comment by Merith-TK** [comment]
Trust me, if i knew why, i would love to fix it,

**Comment by Renegade-Master** [comment]
Unless you have already investigated this, it could be something to do with how permissions were overhauled several times going from Android API 26 - 29, and are being overhauled again for API 30.

If that is the case, then adding a permission request for local/internal storage to the [Android Manifest](https://github.com/Merith-TK/devdocs_webapp_kotlin/blob/master/app/src/main/AndroidManifest.xml) might fix the issue.

If you can't, or understandably don't want to try this, I can give it a go and let you know how it goes when I get some time.

**Comment by Merith-TK** [comment]
If you could, that would be great, just got a new pc last week and I am fighting for space on it, so I don't have the room to install the android sdk,
